tests/print/tests.c

```diff
@@ -130,6 +130,55 @@ static void test_print_simple(void)
 		rte_pktmbuf_free(packets[i]);
 }
 
+
+static void test_print_pcap(void)
+{
+	int i;
+	struct pg_brick *gen, *print, *col;
+	struct pg_error *error = NULL;
+	struct rte_mbuf *packets[NB_PKTS];
+	struct rte_mbuf **pkts;
+	uint64_t pkts_mask;
+	FILE *output = fopen("tests.pcap", "w+");
+
+	g_assert(output);
+	build_packets(packets);
+	gen = pg_packetsgen_new("gen", 1, 1, EAST_SIDE, packets, NB_PKTS,
+				&error);
+	g_assert(!error);
+	print = pg_print_new("My print", 1, 1, output, PG_PRINT_FLAG_PCAP, NULL,
+			     &error);
+	g_assert(!error);
+	col = pg_collect_new("col", 1, 1, &error);
+	g_assert(!error);
+
+	pg_brick_link(gen, print, &error);
+	g_assert(!error);
+	pg_brick_link(print, col, &error);
+	g_assert(!error);
+
+	pg_brick_burst_to_east(gen, 0, packets,
+			       pg_mask_firsts(NB_PKTS), &error);
+	g_assert(!error);
+
+	pkts = pg_brick_west_burst_get(col, &pkts_mask, &error);
+	g_assert(!error);
```

Comment:
Maybe check pkts_mask result ?
And you should check that tests.pcap exists, have some data and delete it at the end of test